mf4_csv/main.py

```diff
@@ -0,0 +1,52 @@
+
+import os 
+from asammdf import MDF, Signal
+from os import walk
+import re
+import glob
+
+
+output_dir = "./output_in_csv"
+
+def prepare_dir(mf4_file):
+    my_str = str(mf4_file)
+    my_str = my_str.lower().strip(".mf4")
+    my_str = my_str.replace(".MF4", ".csv")
+    # chars = "./-\"
+    # for c in chars:
+    #     text = text.replace(c, "\\" + c)
+    rx = re.compile('([-./\ ])')
+    my_str = rx.sub('_', my_str)
+    # print(my_str)
+    return my_str
+
+   
+
+def convertMF4toCSV(mf4_file):
+    data = MDF(mf4_file)
+    # f_out = f_in.replace(".MF4", ".csv") 
+    # mdf = MDF(f_in) #read mf4 data 
+
+    # d_dir = prepare_dir(mf4_file)
```

Comment:
Why is the directory preparation commented out?

Reply:
added